<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGame</class>
 <widget class="QWidget" name="NewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Game</string>
  </property>
  <widget class="QFrame" name="MainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>391</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>391</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>301</width>
     <height>391</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="DecissionButtonsFrame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="OKButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CancelButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ObjectFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="whichGameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>game</string>
     </property>
    </widget>
    <widget class="QComboBox" name="whichGameBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="boardSizeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>boardSize</string>
     </property>
    </widget>
    <widget class="QComboBox" name="boardSizeBox_w">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="mctsSimsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MCTSSims</string>
     </property>
    </widget>
    <widget class="QLabel" name="weightLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>weight</string>
     </property>
    </widget>
    <widget class="QComboBox" name="weightBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="aiTurnLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AI first</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="aiTurnBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="mctsSimsBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
    <widget class="QComboBox" name="boardSizeBox_h">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
